src/main/java/com/snowflake/kafka/connector/internal/InternalUtils.java

```diff
@@ -30,7 +30,8 @@ class InternalUtils {
   static final String JDBC_SSL = "ssl";
   static final String JDBC_SESSION_KEEP_ALIVE = "client_session_keep_alive";
   static final String JDBC_WAREHOUSE = "warehouse"; // for test only
-
+  static final String JDBC_AUTHENTICATOR = "authenticator";
```

Comment:
use JDBC enum instead of our own defined string. 

i know this is not the norm in this file but lets use it if we can.

Reply:
Fixed.